### ISSUE TITLE
nushell: × Automatically spreading lists is deprecated

### ISSUE THREAD

**Issue by obfuscated-loop** [ajeetdsouza/zoxide#693]
I just installed `zoxide` from the AUR and added to both my env and config file.

Opening a nushell returns this: 
```
Error:   × Automatically spreading lists is deprecated
    ╭─[/home/j/.zoxide.nu:25:1]
 25 │ # Jump to a directory using only keywords.
 26 │ def-env __zoxide_z [...rest:string] {
    ·                    ────────┬───────
    ·                            ╰── Spreading lists automatically when calling external commands is deprecated and will be removed in 0.91.
 27 │   let arg0 = ($rest | append '~').0
    ╰────
  help: Use the spread operator (put a '...' before the argument)


Error: nu::shell::external_command

  × External command failed
    ╭─[/home/j/.zoxide.nu:25:1]
 25 │ # Jump to a directory using only keywords.
 26 │ def-env __zoxide_z [...rest:string] {
    · ───┬───
    ·    ╰── did you mean 'let-env'?
 27 │   let arg0 = ($rest | append '~').0
    ╰────
  help: No such file or directory (os error 2)
```

I am using nushell version `0.90.1` and zoxide `0.9.2`. I am also new to nushell but I can try a PR if it's appreciated.

**Comment by obfuscated-loop** [comment]
I just noticed that the latest release of zoxide is `0.9.3` - I will give that a try.

**Comment by obfuscated-loop** [comment]
I see this was fixed in #685 - closing now.
